### PR TITLE
automation framework and banner test added

### PR DESCRIPTION
**Pull request by Vamshi-Muniganti**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-282
https://tools.hmcts.net/jira/browse/EUI-188

### Change description ###

Added Automation Framework to XUI Approve Organisation repo and added test for banner checking organisations

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```

### CODE REVIEW

File: test/e2e/support/timeout.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// timeout.js
+var { setDefaultTimeout } = require("cucumber");
+
+setDefaultTimeout(600 * 1000);
```
Comment (andywilkinshmcts) [inline test/e2e/support/timeout.js]: setting to 10 mins here and 1 min in world.js is this correct?

File: test/e2e/support/world.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        throw new Error('selector cannot contain spaces');
+    }
+    return selector.split('|')
+        .map(level => level.split('.')
+            .map(part => processRecursive(part) || `[data-selector~="${part}"]`)
+            .join(''))
+        .join(' ');
+}
+
+const seconds = n => n * 1000;
```
Comment (andywilkinshmcts) [inline test/e2e/support/world.js]: this function doesn't appear to be used, though it could be

File: test/e2e/support/timeout.js (hunk 1 above)
Comment (Vamshi-Muniganti) [reply test/e2e/support/timeout.js]: these values are not used world.js and timeout.js, as we used the same old files which we will be refactoring in our tidyup tasks

File: test/e2e/support/world.js (hunk 2 above)
Comment (Vamshi-Muniganti) [reply test/e2e/support/world.js]: these values are not used world.js and timeout.js, as we used the same old files which we will be refactoring in our tidyup tasks

**Pull request merged**